src/pynwb/__init__.py

```diff
@@ -12,13 +12,48 @@
 from hdmf.backends.hdf5 import HDF5IO as _HDF5IO
 from hdmf.build import BuildManager, TypeMap
 import hdmf.common
+from hdmf.common import load_type_config as hdmf_load_type_config
+from hdmf.common import get_loaded_type_config as hdmf_get_loaded_type_config
+from hdmf.common import unload_type_config as hdmf_unload_type_config
+
 
 CORE_NAMESPACE = 'core'
 
 from .spec import NWBDatasetSpec, NWBGroupSpec, NWBNamespace  # noqa E402
 from .validate import validate  # noqa: F401, E402
 
 
+CUR_DIR = os.path.dirname(os.path.realpath(__file__))
+path_to_config = os.path.join(CUR_DIR, 'config/nwb_config.yaml')
```

Comment:
How does this work if someone has run `pip install pynwb`? Where does the config file live? In that case, it probably makes sense as a file in the user's home directory like `~/.pynwb/nwb_config.yaml`

Reply:
Alternatively, we could specify no default, and the user must supply a path to a configuration file.

Reply:
I will remove the default.